documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+
+Once the paramount objections are addressed, we will consider the proposal to be
+accepted. The team as a whole can move on the next step of the project, whether
+that is implementation planning or implementation itself.
+
+## Outcomes
+
+There are two possible outcomes for any proposal that follows this process:
+ratification or tabling. All raised proposals should eventually be marked as
+ratified (if the decision-making process warrants it) or tabled.
+
+### Ratification
+
+Ratification occurs once all paramount objections have been addressed and the
+author is satisfied with the state of the proposal. At this point, if there are
+issues to create to implement the proposal, they should be created and linked in
+the proposal text relevant to the issue.
+
+### Tabling
+
+Tabling can occur in the following cases:
+
+- At any time, at the discretion of the author
+- Before a proposal is written, if the team reneges on a requested discussion
+- After a proposal is written, but before the discussion happens, if it is clear
+  the proposal will not fit the needs of the project
+- During the objection round when insurmountable paramount objections are raised
+
+In the final case, the participants of the discussion are responsible for
+deciding whether a new proposal should be requested.
+
+In any case, the proposal (if one exists) or the issue (if a proposal does not
+exist) should be updated to reflect the tabled status and the reason why it was
+tabled. It should also indicate whether a new proposal is requested. If a
+proposal exists, the author is tasked with this responsibility. If no proposal
+yet exists, then someone must volunteer to document the reasons for this
+outcome.
+
+## Additional practices
+
+The following additional conventions will help our team make expedient decisions
+while still following the formalised process laid out above.
+
+### Discussion Dashboard
+
+The
+[discussion dashboard is implemented as a GitHub project](https://github.com/orgs/WordPress/projects/79/views/1)
+for tracking the different rounds of a proposal. Proposal authors are expected
+to correctly add and update proposals as they move through the different rounds
+of discussion. This dashboard is used to
+[help ensure we do not assign contributors to more than two discussions in parallel](#minimising-parallel-discussions),
+so it is imperative that we maintain its accuracy.
+
+Each discussion should be tracked as a single card, with issues removed from the
+"Pending proposal" column if a GitHub PR is opened. If a GitHub discussion is
+used for the proposal, because those cannot be represented in GitHub project
+boards, the issue should be used to track the status through the columns.
+
+This dashboard also serves as a record of decisions made on the team since the
+dashboard's inception. Historical decisions that happened before the dashboard
+started being used are not documented there.
+
```

Comment:
I've added a couple of views to track everyone's load:
<img width="1088" alt="Screenshot 2023-03-10 at 3 47 54 PM" src="https://user-images.githubusercontent.com/15233243/224320162-74687645-b88b-4b5c-b7d9-aeb6263675ae.png">

I feel that it would be useful, and hope they do not make the board too crowded. What do you think, @sarayourfriend ?

Reply:
Oh, and I added my open proposal :)